src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -60,8 +60,6 @@ private static DeviceDataManager createDeviceDataManager() {
 		final DeviceConfig apCfgC = new DeviceConfig();
 		apCfgA.boundary = 500;
 		apCfgA.location = List.of(408, 317);
-		apCfgA.allowedTxPowers = new HashMap<>();
-		apCfgA.allowedTxPowers.put(BAND, List.of(29, 30));
```

Comment:
Not too important, but I was curious why this was ever needed.

Reply:
I don't think it ever was, which is why I removed it. It appears that we should only be setting tx powers which are allowed by this list, but since the algorithm doesn't do that (as far as I can tell), I didn't want to give the impression that we did and/or that we properly tested it, so I removed it altogether. I have it on my list of things to look into down the line.